CRM/Price/BAO/LineItem.php

```diff
@@ -605,4 +604,466 @@ public static function getLineItemArray(&$params, $entityId = NULL, $entityTable
     }
   }
 
+  /**
+   * Function to update related contribution of a entity and
+   *  add/update/cancel financial records
+   *
+   * @param array $params
+   * @param int $entityID
+   * @param int $entity
+   * @param int $contributionId
+   * @param $feeBlock
+   * @param array $lineItems
+   * @param $paidAmount
+   *
+   */
+  public static function changeFeeSelections(
+    $params,
+    $entityID,
+    $entity,
+    $contributionId,
+    $feeBlock,
+    $lineItems,
+    $paidAmount
+  ) {
```

Comment:
Reason behind this change is fn is called in only two places and I changed the parameter order as per the lineitem table column.